### PR TITLE
Extended support for messaging-activemq

### PR DESCRIPTION
**Pull request by falle1234Akait**
Added server-properties.
Changed connection-factory->connectors to list instead of string
Added modifying properties on address-settings

### CODE REVIEW

File: plugins/modules/jcliff.py [hunk 1]
```diff
@@ -854,7 +869,7 @@
               connectors:
                 description:
                   - Legacy entries
-                type: str
+                type: list
```
Comment (guidograzioli) [inline plugins/modules/jcliff.py:872]: please add `elements: str` to make ansible-test sanity happy

File: plugins/modules/jcliff.py [hunk 2]
```diff
@@ -1341,7 +1380,7 @@ def main():
                                         type='list', required=False, elements='dict', options=dict(
                                             name=dict(type='str', required=True),
                                             entries=dict(type='list', required=True),
-                                            connectors=dict(type='str', required=False),
+                                            connectors=dict(type='list', required=False),
```
Comment (guidograzioli) [inline plugins/modules/jcliff.py:1383]: `, elements='str'` for same reason as above

**Review by guidograzioli — CHANGES_REQUESTED** [review]
please address the element type change in the two places I commented on

**Review by falle1234Akait — COMMENTED** [review]
Hope I have done everything correctly. Otherwise let me know and I'll try to resolve it.

File: plugins/modules/jcliff.py [hunk 3]
```diff
@@ -772,6 +772,21 @@
           - Create messaging activemq.
         type: dict
         suboptions:
+          server_property:
+            description:
+              - Creates and sets messaging-activemq server properties.
+              type: list
```
Comment (guidograzioli) [inline plugins/modules/jcliff.py:778]: Looking at the sanity test CI failing I believe you have an indent problem here (type and suboptions  same-level with description?)

Comment (falle1234Akait) [reply plugins/modules/jcliff.py]: I have now fixed the indentation. 
I also noticed I hadn't set elements: dict on server_property.

**Review by guidograzioli — APPROVED** [review]
Great contribution, thanks!

**Pull request merged**